server.js

```diff
@@ -20,4 +21,46 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
+app.get("/messages/search", (req, res) => {
+  const { term } = req.query;
```

Comment:
Always nice to try and cover off edge cases and handle them; e.g. what if `term` was an empty string, or undefined? How would you handle that?